insights/client/config.py

```diff
@@ -586,11 +586,11 @@ def _load_config_file(self, fname=None):
         parsedconfig = ConfigParser.RawConfigParser()
         try:
             parsedconfig.read(fname or self.conf)
-        except ConfigParser.Error:
+        except ConfigParser.Error as e:
             if self._print_errors:
                 sys.stdout.write(
-                    'ERROR: Could not read configuration file, '
-                    'using defaults\n')
+                    'ERROR: {0}.\nCould not read configuration file, '
```

Comment:
@vkrizan 

As per the changes the error message will look like below,

Example:
```
ERROR: ValueError. Could not read configuration file, using defaults
```

How about we remove 'ERROR' so that the message will be as below?

```
ValueError. Could not read configuration file, using defaults
```

Reply:
Sure we can do that, however not every Exception ends with "Error" to realize it is an error message.

Reply:
Too soon. :slightly_smiling_face:  The base exception this is catching has all child exception ending with Error: https://docs.python.org/3/library/configparser.html#exceptions